Makefile.common

```diff
@@ -42,12 +44,15 @@ ifneq ($(DISTFILES_MIRROR),)
 	URL := $(addprefix $(DISTFILES_MIRROR)/,$(SRC_FILE))
 endif
 
+shellquote = '$(subst ','\'',$1)'
+
 get-sources: $(SRC_FILE)
 
 $(SRC_FILE):
 ifneq ($(SRC_FILE), None)
-	@wget -q $(URL)
-	@wget -q $(addsuffix .asc,$(URL))
+	$(if $(FETCH_CMD),,$(error You can not run this Makefile without having FETCH_CMD defined))
+	$(foreach a,$(URL),$(FETCH_CMD) $(call shellquote,$(notdir $a)) $(call shellquote,$a) && ):
+	$(foreach a,$(URL),$(FETCH_CMD) $(call shellquote,$(notdir $a).asc) $(call shellquote,$a.asc) && ):
```

Comment:
This is a bit too complex. Maybe split into separate targets for each file? Like it's done in vmm-xen-stubdom-legacy.